Dockerfile

```diff
@@ -1,18 +1,23 @@
 FROM alpine:edge
 
+ENV K8S_VERSION=v1.6.3
```

Comment:
We should keep in mind then that any `docker build` might change the `kubectl` version and therefor create a different image. Are we OK with this? It should be fine I guess, I just wanted to make sure that we're all on the same page.